client/trino-jdbc/src/main/java/io/trino/jdbc/ConnectionProperties.java

```diff
@@ -461,6 +463,15 @@ public ExternalAuthenticationTimeout()
         }
     }
 
+    private static class ExternalAuthenticationSharedToken
+            extends AbstractConnectionProperty<Boolean>
+    {
+        public ExternalAuthenticationSharedToken()
+        {
+            super("externalAuthenticationSharedToken", Optional.of("false"), NOT_REQUIRED, ALLOWED, BOOLEAN_CONVERTER);
```

Comment:
It looks to me more about implementation than UX behaviour.

How about `externalAuthenticationCache=NONE|MEMORY`? Where `NONE` is default.